charts/kubetemplates/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Kubernetes
 name: kubetemplates
-version: 1.19.0-beta.10
+version: 1.19.0-beta.11
```

Comment:
Let's bump it to `1.25.0-beta.1` . If `1.25.0` is the k8s version for which we are making this changes.

Reply:
updated version.